universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -124,6 +128,16 @@ public Tag renderProgramListItem(
         div(programDescriptionText)
             .withClasses(Styles.TEXT_GRAY_700, Styles.TEXT_BASE, Styles.MB_8, Styles.LINE_CLAMP_3);
 
+    Tag programDeepLink =
+        input()
```

Comment:
Why use an input for this? Is it better for copying?

Reply:
Yeah, just a bigger click target for double-clicking to highlight